posts/2021/intake-esm-tutorial-2021.md

```diff
@@ -0,0 +1,50 @@
+---
+author: Max Grover
+date: 2021-12-08
+tags: python-tutorial-series, data-access, xarray, intake
+---
+
+# Intake-ESM Tutorial
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ProjectPythiaTutorials/intake-esm-tutorial_2021_12_08/main)
+
+The Project Pythia Python Tutorial Seminar Series continues with the Intake-ESM Tutorial on **Wednesday, December 8 at 1 PM Mountain Standard Time**. This session will be led by [Max Grover](https://github.com/mgrover1).
+
+The content for this tutorial is hosted in the `.ipynb` notebook file(s) within the `tutorial` directory of this repository.
+
+## Bio
+
+Max works as software engineer within the Climate and Global Dynamics Lab here at NCAR. He graduated with a B.S in meteorology from [Valparaiso University](https://www.valpo.edu/geography-meteorology/) in 2019 and will complete his M.S. in atmospheric science this spring at the [University of Illinois Urbana-Champaign](https://atmos.illinois.edu/).
+
+He works on Python-based tools to support climate model evaluation and contributes to the Earth System Data Science initiative. His research in graduate school focused on using observational data collected during the RELAMPAGO-CACTI campaign in Argentina to study mesoscale convective dynamics, primarily utilizing mobile radar observations as well as GOES-16 satellite imagery. This is not his first time working at UCAR – he took part in the Unidata summer internship program in 2019, working with Ryan May on contributing METAR parsing functionality to MetPy, which motivated him to pursue a career in software engineering.
+
+He enjoys hiking, visiting breweries, and spending time on the water (his summer job, prior to graduate school, was at a resort in southern Wisconsin where he worked as a tour boat captain on Delavan Lake). He is also a big fan of Hawaiian shirts, with a collection of more than 40.
+
+## How-to-Run
+
+**All of the tutorials in this organization can be followed by clicking the "Launch Binder" button at the top level or in each directory.**
+
+You may also download the environment to run locally:
+
+- Check your conda install with `conda --version` and if necessary [install by following these instructions](https://docs.conda.io/en/latest/miniconda.html).
+
+- Clone this repository by running `git clone https://github.com/ProjectPythia/intake-esm-tutorial_2021_12_08` in your terminal.
```

Comment:
Should the path be ProjectPythiaTutorials/intake-esm-tutorial_2021_12_08 ?

Reply:
Ahh yes!